requirements.txt

```diff
@@ -0,0 +1,2 @@
+paramiko
```

Comment:
It would be a good idea to lock the version of the dependency so the project doesn't break when any of those gets updated on a backwards incompatible way